index.html

```diff
@@ -91,11 +91,11 @@
 	}
     </style>
 
-<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" type="text/css"></link>
-<link rel="stylesheet" href="//yandex.st/highlightjs/7.3/styles/github.min.css" type="text/css"></link>
-<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
-<script type="text/javascript" src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
-<script type="text/javascript" src="//yandex.st/highlightjs/7.3/highlight.min.js"></script>
+<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" type="text/css">
```

Comment:
Also the page seemed to load alright for me via rawgit. @hikaruit15 Could you try it again?